### PR TITLE
updated tiny_fMRI_inteference to use setup.m

### PR DESCRIPTION
**Pull request by stevevanhooser**
Small change (in title)

### CODE REVIEW

File: setup.m
Comment (theavuik) [inline setup.m:1]: Maybe also fix the typo exmamples to examples?

Comment (theavuik) [reply setup.m:1]: For the rest I am OK.

Comment (stevevanhooser) [reply setup.m:1]: fixed it

Comment (stevevanhooser) [reply setup.m:1]: good find

**Review by theavuik — APPROVED** [review]

**Pull request merged**